src/EnduranceTests/NServiceBus.AzureServiceBus.EnduranceTests.csproj

```diff
@@ -2,12 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net452</TargetFramework>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
This project was not used throughout the entire ASBv7... we might remove it as well.